src/Serializers/ContractBytecodeSerializer.js

```diff
@@ -0,0 +1,34 @@
+const RLP = require('rlp')
+const BaseSerializer = require('./BaseSerializer')
+const BytecodeSerializer = require('./BytecodeSerializer')
```

Comment:
I propose to combine BytecodeSerializer and this one because they are not used separately.

Reply:
Well, `BytecodeSerializer` is actually the bytecode serializer, this one inherits it's bad name from the ERL codebase, as this is not a serializer at all, but more or less contract structure encoder.

Although you're right it is not used alone right now, I prefer to keep them separate for readability